volatility3/framework/plugins/windows/envars.py

```diff
@@ -71,13 +71,25 @@ def _get_silent_vars(self) -> List[str]:
                 sys = hive.get_key(
                     "CurrentControlSet\\Control\\Session Manager\\Environment"
                 )
-            except (KeyError, registry.RegistryFormatException):
-                with contextlib.suppress(KeyError, registry.RegistryFormatException):
+            except (
+                KeyError,
+                registry.RegistryFormatException,

```

Comment:
Should both of these exceptions inherit from a top level exception (`registry.RegistryException`) that then is made more specific?  Seems like it'll be useful in most instances?

Reply:
Yeah, I had wondered about that too when I started making the changes.  It'd definitely be cleaner, I can switch it over to RegistryException

Reply:
Sounds like a plan, thanks.  5:)